.gitignore

```diff
@@ -0,0 +1,3 @@
+
+.env
```

Comment:
Very nitpicky: I would prefer not to have such a file that caters for a local workflow of a certain dev setup. The single artifact this repo is expected to produce is a Docker image and there is no need to ignore it. Files that are specific to a dev machine should go into its global gitignore.